docs/transactions.md

```diff
@@ -1430,20 +951,15 @@ If more than 1 Trustee signature is required to add the account, the account
 will be in a pending state until sufficient number of approvals is received.
 
 - Parameters:
-    - `address`: string // account address; bech32 encoded
-    - `pub_key`: string // account public key; bech32 encoded
-    - `vid`: 16 bit number // vendor id (only needed for vendor role)
-    - `roles`: array<string> // the list of roles to assign to account 
-- In State:
-  - `auth` store  
-  - `1:<address>` : `<account info> + <list of approvers>`
-  - `2:<address>` : `<account info>` (if just 1 Trustee is required)  
+    - address: `string` - account address; Bench32 encoded
+    - pub_key: `string` - account public key; Bench32 encoded
```

Comment:
```suggestion
    - pub_key: `string` - Account's Protobuf JSON encoded public key
```

Reply:
I think account's public key is a bench32 encoded key, so it seems a correct description

Reply:
While testing found that it needed full JSON encoded public key.

Reply:
Yes, you are right, thank you. Fixed.